src/jit/CMakeLists.txt

```diff
@@ -168,10 +168,15 @@ endif()
 
 set(CLR_EXPORTED_SYMBOL_FILE ${CLRJIT_EXPORTS_DEF})
 
+set(JIT_BASE_NAME clrjit)
+if (CLR_BUILD_JIT32)
+  set(JIT_BASE_NAME ryujit)
+endif()
```

Comment:
And if I understand the description of what we're doing, only the clrjit.dll would be put into the platform package.  So consumers of the package would only ever see the clrjit name.